### PR TITLE
Update the information, pictures and added videos

### PR DESCRIPTION
**Pull request by ivanha09**
Originally was just going to add videos, but there was no fix issue section and I thought changing some of the information to reflect the current status of Sider would be useful.

### CODE REVIEW

File: docs/getting-started/working-with-issues.md [hunk 1]
```diff
@@ -7,42 +7,55 @@ hide_title: true
 
 # Working with Issues
 
-You can view all the issues Sider has found on a given pull request from the pull request page in Sider. This page also offers several handy ways to interact with issues, including ignoring issues and requesting feedback.
+Although Sider needs all issues to be dealt with before allowing commit status on GitHub, this doesn't mean that all issues need to be fixed. A combination of fixing or closing all issues will also allow commit status on GitHub.
```
Comment (wata727) [inline docs/getting-started/working-with-issues.md]: I'm wondering if "allowing commit status" is a proper expression :thinking: 
According to [GitHub Help](https://help.github.com/en/articles/about-required-status-checks), they say "passing status checks". What do you think?

File: docs/getting-started/working-with-issues.md [hunk 2]
```diff
@@ -7,42 +7,55 @@ hide_title: true
...
+ <iframe src="https://www.youtube.com/embed/16MuYzj_Ml0" frameborder="0" allowfullscreen
+   style="position:absolute;top:0;left:0;width:100%;height:100%;"></iframe>
+</div>
 
 ## Requesting Feedback
-If you are not sure whether an issue is a false positive or not, you can ask your teammates to help you figure it out. Click the `Comment` button to post a comment to GitHub.
+If you are not sure whether an issue is a false positive or not, you can ask your teammates to help you figure it out. Click the `Comment on GitHub` button to post a comment to GitHub.
 
-![Issue comments](../assets/issues-comment.png)
+![Issue comments](../assets/issues-comment-v2.png)
```
Comment (wata727) [inline docs/getting-started/working-with-issues.md]: Can we remove these unused images? :upside_down_face:

File: docs/getting-started/working-with-issues.md (hunk 1 above)
Comment (ivanha09) [reply docs/getting-started/working-with-issues.md]: @wata727 Thanks! I'll change it to the proper GitHub expression.

File: docs/getting-started/working-with-issues.md (hunk 2 above)
Comment (ivanha09) [reply docs/getting-started/working-with-issues.md]: @wata727 Sure. I wasn't sure if they are used in other parts of the document so I didn't want to just replace them.

Comment (wata727) [reply docs/getting-started/working-with-issues.md]: In my experience, these images are used in only one place. I'm glad if you remove these ones.

**Pull request merged**